Options.lua

```diff
@@ -112,43 +112,43 @@ function frame:CreateOptions()
 	autoSellSoulboundOnly:SetPoint("TOPLEFT", title, "BOTTOMLEFT", 120, -205)
 	autoSellSoulboundOnly:SetScript("PostClick", function(self, button, down)
 		SoulboundOnly = self:GetChecked()
-		ns.markWares()
+		Peddler.markWares()
```

Comment:
Why was this function public? How could I use this in my own addon?

Reply:
Ah, I thought I needed this to be able to access markWares in another file.

Reply:
Ho, I understand now, thanks.